docs/supported-libraries.md

```diff
@@ -92,7 +92,7 @@ These are the supported libraries and frameworks:
 | [JSP](https://javaee.github.io/javaee-spec/javadocs/javax/servlet/jsp/package-summary.html)                                                 | 2.3+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Controller Spans [3]                                                                                                                    |
 | [Kotlin Coroutines](https://kotlinlang.org/docs/coroutines-overview.html)                                                                   | 1.0+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Context propagation                                                                                                                     |
-| [Ktor](https://github.com/ktorio/ktor)                                                                                                      | 1.0+                               | [opentelemetry-ktor-1.0](../instrumentation/ktor/ktor-1.0/library),<br>[opentelemetry-ktor-2.0](../instrumentation/ktor/ktor-2.0/library)                                                                                                                                                                                                                                               | [HTTP Client Spans], [HTTP Client Metrics], [HTTP Server Spans], [HTTP Server Metrics]                                                  |
+| [Ktor](https://github.com/ktorio/ktor)                                                                                                      | 1.0+                               | [opentelemetry-ktor-1.0](../instrumentation/ktor/ktor-1.0/library),<br>[opentelemetry-ktor-2.0](../instrumentation/ktor/ktor-2.0/library) (supports Ktor 3.*)                                                                                                                                                                                                                           | [HTTP Client Spans], [HTTP Client Metrics], [HTTP Server Spans], [HTTP Server Metrics]                                                  |
```

Comment:
I think you can remove `(supports Ktor 3.*)`. Unless stated otherwise instrumentation for a version `x` is expected to also cover following versions.

Reply:
```suggestion
| [Ktor](https://github.com/ktorio/ktor)                                                                                                      | 1.0+                               | [opentelemetry-ktor-1.0](../instrumentation/ktor/ktor-1.0/library),<br>[opentelemetry-ktor-2.0](../instrumentation/ktor/ktor-2.0/library)                                                                                                                                                                                                                           | [HTTP Client Spans], [HTTP Client Metrics], [HTTP Server Spans], [HTTP Server Metrics]                                                  |
```